src/main/java/mate/academy/hibernate/relations/service/impl/MovieServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import java.util.Optional;
+import mate.academy.hibernate.relations.dao.impl.MovieDaoImpl;
 import mate.academy.hibernate.relations.model.Movie;
 import mate.academy.hibernate.relations.service.MovieService;
 
 public class MovieServiceImpl implements MovieService {
+    private MovieDaoImpl movieDao;
+
+    public MovieServiceImpl(MovieDaoImpl movieDao) {
+        this.movieDao = movieDao;
+    }
+
     @Override
     public Movie add(Movie movie) {
-        return null;
+        return movieDao.add(movie);
     }
 
     @Override
     public Movie get(Long id) {
-        return null;
+        Optional<Movie> optionalMovie = movieDao.get(id);
+        return optionalMovie.get();
```

Comment:
Directly calling `get()` on the `Optional` object without checking if it's present can lead to a `NoSuchElementException` if the `Optional` is empty. It's better to use `orElseThrow` with a meaningful exception to handle the case when the `Movie` with the given `id` is not found.